### PR TITLE
Added db changelog for RDM-9696

### PR DESCRIPTION
**Pull request by sumansreenivasan**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDM-9696

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -137,7 +137,7 @@ ccd.draft.encryptionKey=${CCD_DRAFT_ENCRYPTION_KEY:xxxxxxxxxxxxxxxx}
 ccd.draft.maxTTLDays=${CCD_DRAFT_TTL_DAYS:180}
 
 # Search
-search.elastic.enabled=${ELASTIC_SEARCH_ENABLED:false}
+search.elastic.enabled=${ELASTIC_SEARCH_ENABLED:true}
```
Comment (MSancaktutar) [inline src/main/resources/application.properties:140]: why change this?

Comment (sumansreenivasan) [reply src/main/resources/application.properties]: Reverted the changes. It was by mistake.

File: src/main/resources/db/changelog/db.changelog-RDM-9696.xml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+    <changeSet id="rdm-9696" runInTransaction="false" author="suman.sreenivasan@hmcts.net">
+        <sql dbms="postgresql"
+             endDelimiter="\nGO"
+             splitStatements="true"
+             stripComments="true">
+            CREATE INDEX CONCURRENTLY idx_case_data_languagePreferenceWelsh ON public.case_data USING btree (btrim(upper((data #>> '{languagePreferenceWelsh}'::text[]))));
```
Comment (MSancaktutar) [inline src/main/resources/db/changelog/db.changelog-RDM-9696.xml:11]: each index should be created if it doesn't exist.

Comment (sumansreenivasan) [reply src/main/resources/db/changelog/db.changelog-RDM-9696.xml]: changed the query.

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Review by rajaram-nagarajan — APPROVED** [review]
LGTM

**Pull request merged**